index.html

```diff
@@ -0,0 +1,191 @@
+        <li>1 pizca pimiento rojo en julianas</li>
+        <li>1 taza champiñón en rebanadas</li>
+        <li>1 diente ajo finamente picado</li>
+        <li>1 pizca sal</li>
+      </ul>
+
+    </figure>
+
+    <figure class="video">
+      <iframe class="youtube" width="480" height="315" src="https://www.youtube.com/embed/xVyMN7do-2E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      <h3>Panqueque de Banana y avena</h3>
+      <h4>Ingredientes</h4>
+      <ul>
+        <li>1 banana madura</li>
+        <li>2 huevos</li>
+        <li>40 gramos de copo de avena</li>
+        <li>Aceite</li>
+      </ul>
+      
+    </figure>
+  </div>
+
+</div>
+
+<div class="instagram-container" id="insta">
+
+  <div class="instagram">
+
+  <div class="instatexto">
+    <h2>Seguinos por Instagram</h2>
+    <p><strong>Follow</strong> y sumate a nuestros seguidores para no  perderte<br>nuestras recetas semanales, y participar de increíbles premios!</p>
+    <h4><strong>Click</strong> en el logo para seguirnos!</h4>
+    <a href="https://www.instagram.com/holapanzasana/">
+      <img class="logoinsta hvr-pulse-grow" src="Resources/instagram.svg"></a>
+      <div class="underline">   </div>
+  </div> 
+
+  <div class="instafoto">
+    <img class="fotoinsta" src="Resources/insta1.png">
+  </div>
+  </div>
+
+</div>
+
+<div id="formulario">
+  <h1>&bull; Envianos tu receta &bull;</h1>
+  <div class="underline">
+  </div>
+  <div class="icon_wrapper">
+    <img class="icon" src="Resources/recipe.svg">    
+  </div>
+  <form action="#" method="post" id="contact_form">
+    <div class="name">
+      <label for="name"></label>
+      <input type="text" placeholder="Nombre" name="name" id="name_input" required>
+    </div>
+    <div class="email">
+      <label for="email"></label>
+      <input type="email" placeholder="e-mail" name="email" id="email_input" required>
+    </div>
+    <div class="telephone">
+      <label for="name"></label>
+      <input type="text" placeholder="Receta" name="telephone" id="telephone_input" required>
+    </div>
+    <div class="subject">
+      <label for="subject"></label>
+      <select placeholder="Tipo de Receta" name="subject" id="subject_input" required>
+        <option disabled hidden selected>Tipos de Receta</option>
+        <option>Platos</option>
+        <option>Postre</option>
+        <option>Bebidas</option>
+      </select>
+    </div>
+    <div class="message">
+      <label for="message"></label>
+      <textarea name="message" placeholder="Ingredientes" id="message_input" cols="30" rows="5" required></textarea>
+    </div>
+    <div class="submit">
+      <input type="submit" value="Enviar" id="form_button" />
+    </div>
+  </form><!-- // End form -->
+</div>
+
+<footer>
+    <h5>© Panza Sana | Cocina Saludable | Buenos Aires - Argentina</h5>
+</footer> 
+
+<script src="scrollReveal.js"></script>
+</body>
+</html>
```

Comment:
it seems that u should be adding a new line at the end of the file

Reply:
Done!